app/src/main/java/mod/trindadedev/ui/preferences/Preference.java

```diff
@@ -0,0 +1,22 @@
+package mod.trindadedev.ui.preferences;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import com.sketchware.remod.databinding.PreferenceBinding;
+
+public class Preference extends LinearLayout {
+    private PreferenceBinding binding;
+
+    public Preference(Context context, String title, String description, View.OnClickListener listenerClick) {
+        super(context);
+        binding = PreferenceBinding.inflate(LayoutInflater.from(context), this, true);
```

Comment:
Where attributes from layout?
You must use `merge` tag to avoid nesting layouts.